README.rst

```diff
@@ -1,8 +1,22 @@
-``zope.securitypolicy``
-=======================
+=====================
+ zope.securitypolicy
+=====================
+
+.. image:: https://img.shields.io/pypi/v/zope.securitypolicy.svg
+   :target: https://pypi.python.org/pypi/zope.securitypolicy/
```

Comment:
Let's use https://pypi.org/project/... here

Reply:
whoops, copy-and-pasted from zope.authentication, so I'll fix it there too.